src/Behat/Behat/Output/Node/EventListener/Statistics/HookStatsListener.php

```diff
@@ -111,7 +111,7 @@ private function captureHookStat(CallResult $hookCallResult)
     {
         $call = $hookCallResult->getCall();
         $callee = $call->getCallee();
-        $hook = (string) $callee;
+        $hook = $callee->__toString();
```

Comment:
PSalm is not happy with the cast and requires a explicit call to __toString()

Reply:
I ran this with the full level 6 enabled and did not get any complains, so maybe it is detected at another level. Let me try to see which is the best way to handle this

Reply:
I have narrowed the types before applying the conversion to string